flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -116,6 +118,14 @@ else if (file.exists()) {
 			}
 			catch (NoSuchFileException e) {
 				// if the file is already gone (concurrently), we don't mind
+			} catch (AccessDeniedException e) {
```

Comment:
Is it ok to special case this inside the catch block? (I.e for windows check again, otherwise rethrow?